api-reference/admin/webhooks/introduction.mdx

```diff
@@ -0,0 +1,41 @@
+---
+sidebarTitle: "Introduction"
+title: Learn more about Webhooks
+description: Webhooks allow you to get notified on changes
+---
+
+## Events
+Webhooks are an easy way to get informed about changes in your Hanko instance (e.g. user or email updates).
+Hanko provides the following event structure:
+
+| Event                     | Triggers on                                                                                        |
+|---------------------------|----------------------------------------------------------------------------------------------------|
+| user                      | user creation, user deletion, user update, email creation, email deletion, change of primary email |
+| user.create               | user creation                                                                                      |
+| user.delete               | user deletion                                                                                      |
+| user.update               | user update, email creation, email deletion, change of primary email                               |
+| user.update.email         | email creation, email deletion, change of primary email                                            |
+| user.update.email.create  | email creation                                                                                     |
+| user.update.email.delete  | email deletion                                                                                     |
+| user.update.email.primary | change of primary email                                                                            |
+
+As you can see, events can have subevents. You are able to filter which events you want to receive by either selecting
+a parent event when you want to receive all subevents or selecting specific subevents.
+
+## Restrictions
+
+Due to security concerns there are some usage restrictions for webhooks in place:
+
+* A webhook will be disabled after 30 days without usage. In this scenario usage describes the successful triggering of events
+* A webhook will also be disabled when our trigger mechanism is not successful. The mechanism will try 5 attempts of sending events before disabling a webhook. A trigger is unsuccessful when:
+  * your callback URL is not reachable due to network errors or a wrong url
+  * your webhook endpoint returns HTTP Status codes >= 400
+
+The restrictions are not available for webhooks defined in the config file.
+
+## Data
+
+When a webhook is triggered it will send you a **JSON** body which contains the event and a jwt.
+The JWT contains 2 custom claims:
+* **data**: contains the whole object for which the change was made. (e.g.: the whole user object when an email or user is changed/created/deleted)
```

Comment:
I think we can move this to the guides section of main docs, wdyt?

So after migrations, we can have a separate section for webhooks? it would just be an mdx file though (not a separate group)
<img width="458" alt="Screenshot 2024-01-23 at 11 46 08 AM" src="https://github.com/teamhanko/docs/assets/62984427/817bcba6-8553-4eaa-a26d-5f5e408b9376">

Reply:
Hi @shentschel, you seem to have moved the whole webhook api-reference to the guides section as well right? I meant just the introduction part(the one I highlighted here). Sorry if it wasn't clear

Reply:
That was more an oversight on my part, sorry! I moved the api-references back to the api-reference folder.  I also updated the text for the introduction with remarks from Lennart.